q2_types/per_sample_sequences/_format.py

```diff
@@ -14,12 +14,64 @@
 import skbio
 import skbio.io
 import yaml
+import qiime2
 import qiime2.plugin.model as model
 from qiime2.plugin import ValidationError
 
 from ..plugin_setup import plugin
 
 
+# TODO: joined reads
+
+class FastqAbsolutePathManifestFormatX(model.TextFileFormat):
+    """
+    Mapping of sample identifiers to absolute filepaths
+
+    """
+    METADATA_COLUMNS = None
+
+    def _validate_(self, level):
+        # First, ensure it is valid Metadata
+        md = qiime2.Metadata.load(self)
```

Comment:
Probably need to catch any errors raised here and re-raise as a validation error.